aiormq/connection.py

```diff
@@ -286,7 +286,9 @@ def __init__(
         if self.url.path == "/" or not self.url.path:
             self.vhost = "/"
         else:
-            self.vhost = self.url.path[1:]
+            quoted_vhost = self.url.path[1:]
+            # yarl>=1.9.5 skips unquoting backslashes in path
+            self.vhost = quoted_vhost.replace("%2F", "/")
```

Comment:
may be use `urllib.parse.unquote` here?

Reply:
Yarl is already unquoting all chars but ignores just the `%2F` - you can check it out [here](https://github.com/aio-libs/yarl/pull/1057).
I think that there is no reason to unquote all chars again here as it may break other cases.